tests/test_keyboard_shortcuts.py

```diff
@@ -0,0 +1,279 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import random
+
+from pages.treeherder import TreeherderPage
+
+
+def test_close_open_panels(base_url, selenium):
+    """Shortcut: 'esc'
```

Comment:
First line of the docstring should be a summary. See https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings for details.